openassessment/xblock/static/sass/oa/utilities/_shame.scss

```diff
@@ -2,6 +2,10 @@
 // ====================
 
 // NOTES: use for any bad-form/orphaned scss that knowingly violate edX FED architecture/standards (see - http://csswizardry.com/2013/04/shame-css)
+#seq_content .wrapper--xblock .openassessment h3.staff__grade__title {
```

Comment:
@clrux Why did you use an ID on the original rule that Eric is trying to override? I thought we generally avoid IDs for CSS and just rely on classes (especially since IDs can force this sort of ugliness for overriding, and IDs also have uniqueness constraints).